backend/sites/src/app/entities/siteProfileOwners.entity.ts

```diff
@@ -42,15 +42,15 @@ export class SiteProfileOwners {
   })
   agentAuthorizedInd: string | null;
 
-  @ManyToOne(
-    () => SiteProfiles,
-    (siteProfiles) => siteProfiles.siteProfileOwners,
-  )
-  @JoinColumn([
-    { name: 'site_id', referencedColumnName: 'siteId' },
-    { name: 'date_completed', referencedColumnName: 'dateCompleted' },
-  ])
-  siteProfiles: SiteProfiles;
+  // @ManyToOne(
+  //   () => SiteProfiles,
+  //   (siteProfiles) => siteProfiles.siteProfileOwners,
+  // )
```

Comment:
Remove the commented code

Reply:
need this for future in case if there will be conflict while data sync